wherehows-web/app/templates/components/search/search-facet.hbs

```diff
@@ -0,0 +1,32 @@
+<div class="nacho-container search-facet">
+  <section class="search-facet__header">
+    <h4 class="search-facet__title">{{facet.displayName}}</h4>
+    {{#if showClear}}
+      <button
+        class="nacho-button--tertiary"
+        {{action onClear facet}}>
+        clear
+      </button>
+    {{/if}}
+  </section>
+  {{#each facet.values as |facetValue|}}
+    {{#get-yield object=counts property=facetValue.value default=0 as |count|}}
```

Comment:
Mentioned this above, but looking at this get-yield at work on the template, the benefits aren't really that apparent to me? Maybe there's something I'm not seeing?